src/expressions/baseExpression.ts

```diff
@@ -311,6 +311,10 @@ module Plywood {
       return chainVia('multiply', expressions, Expression.ONE);
     }
 
+    static power(expressions: Expression[]): Expression {
+      return chainVia('power', expressions, Expression.ZERO);
+    }
```

Comment:
OMG so much power.

This function is insane if you think about it. It's like somebody will be like: "I have this list of expressions that I want to all be to the power of each other"

![](http://www.reactiongifs.us/wp-content/uploads/2013/06/ill_allow_it_community.gif)